apis/swagger.yml

```diff
@@ -539,14 +539,29 @@ definitions:
         type: "string"
         description: |
           ID of preheat task.
+      status:
+        type: "string"
+        description: "the status of preheat task"
```

Comment:
Could you list the enum value of the status field.:

* waiting
* running
* failed
* success

And we should add more details about that.
Since when a preheat task is finished, I think after some time this object should be GCed, otherwise it will lead to memory leak.